src/bundle/Resources/public/scss/core/_custom.tooltip.scss

```diff
@@ -91,6 +91,7 @@
                 color: $ibexa-color-dark;
                 background-color: $ibexa-color-info;
                 border-color: transparent;
+                box-shadow: 0 calculateRem(4px) calculateRem(6px) rgba(65, 145, 255, 0.1);
```

Comment:
I don't think that we want to use this kind of rgba, we should use one of our palette colors (that's `$ibexa-color-info` )